src/data-extractor/javaDataExtractor.ts

```diff
@@ -47,8 +47,10 @@ export class JavaDataExtractor extends AbstractDataExtractor {
     return { type: variable.type, name: variable.name, value: variable.value };
   }
 
-  createVariableId(variable: DebugProtocol.Variable): string {
-    return this.isNullVariable(variable) ? addNullPrefix(hash(variable)) : addObjectPrefix(variable.value.split(this.sizeSuffix)[0]);
+  createVariableId(variable: DebugProtocol.Variable, parentId?: string | undefined): string {
```

Comment:
```suggestion
  createVariableId(variable: DebugProtocol.Variable, parentId?: string): string {
```
The ? implies that it can be undefined.